test/www/jxcore/test/pkcs12foldergood/.gitignore

```diff
@@ -0,0 +1,3 @@
+﻿# The folder where a good pkcs12 file will be saved.
+
+!.gitignore
```

Comment:
I'm confused. Why would you need to tell the system not to ignore .gitignore? It wouldn't ignore it anyway?